doc/user/layouts/shortcodes/private-preview.html

```diff
@@ -6,8 +6,8 @@
   It isn't subject to our backwards compatibility guarantees.
   {{if not (.Get "enabled-by-default")}}
   <br><br>
-  You must
-  <a href="https://materialize.com/contact/">contact us</a>
-  to enable this feature in your Materialize region.
+  To enable this feature in your Materialize region,
+  <a href="https://materialize.com/docs/support/">contact your Materialize
```

Comment:
The "your" here is orphaned. I think we should either say "contact Materialize support" or "contact your Materialize support contact."

The former feels preferable?

Reply:
Nice spot, Aru! We should be consistent with the rest of the documentation, where we...mostly use "contact our team".

Reply:
Definitely happy to converge but just making sure that we're not erasing a distinction where in some cases, we want our customers to contact us per their usual support channel and other cases, where we want people to contact us to sign up to schedule a demo or guided trial or whatnots.  

Basically, is it: a) maintain a distinction between general reachouts vs support through the wording or b) just go with the same wording but rely on the specific links to do the differentiating once they click. 

If this hasn't been an issue, can definitely go with "contact our team" if that's been what we've mostly been using.

Reply:
At least the ones I'm referring to are used in the same context (pointing to `/support`).